ext/js/data/options-util.js

```diff
@@ -1348,6 +1350,17 @@ export class OptionsUtil {
         await this._applyAnkiFieldTemplatesPatch(options, '/data/templates/anki-field-templates-upgrade-v41.handlebars');
     }
 
+    /**
+     * - Added scanning.scanAltText
+     * @type {import('options-util').UpdateFunction}
+     * @param {import('settings').Options} options
```

Comment:
this param annotation shouldn't be there i think

Reply:
Why not? If anything I was confused why it was `any` in all the other upgrade functions, could it break things by being typed due to different versions of the settings maybe?